server/drivers/hd44780.c

```diff
@@ -258,7 +253,6 @@ static const char *backlight_type_str(int backlight_type)
 			strcat(buffer, bl_value_mapping[i].name);
 			unknown &= ~bl_value_mapping[i].value;
```

Comment:
Actually you want `unkown |= bl_value_mapping[i].value;`?

Reply:
No, it's OK, just `unknown` shall be initialized to `backlight_type`. I meant to print unknown bits if there are any.